src/redis-benchmark.c

```diff
@@ -1614,7 +1614,8 @@ int parseOptions(int argc, char **argv) {
 " -a <password>      Password for Redis Auth\n"
 " --user <username>  Used to send ACL style 'AUTH username pass'. Needs -a.\n"
 " -u <uri>           Server URI.\n"
-" -c <clients>       Number of parallel connections (default 50)\n"
+" -c <clients>       Number of parallel connections (default 50). Note: number of\n"
+"                    clients has to be the same or higher than the amount of nodes.\n"
```

Comment:
This only makes sense if `--cluster` is used, right? Maybe mention that?

I think "number of nodes" is better words than "amount of nodes".

Reply:
@zuiderkwast Ok I have just addressed your comment. thanks.